rust/main/chains/hyperlane-ethereum/src/contracts/validator_announce.rs

```diff
@@ -150,7 +150,11 @@ where
     }
 
     #[instrument(ret, skip(self))]
-    async fn announce_tokens_needed(&self, announcement: SignedType<Announcement>) -> Option<U256> {
+    async fn announce_tokens_needed(
+        &self,
+        announcement: SignedType<Announcement>,
+        chain_signer: Option<String>,
```

Comment:
my other suggestion would've been to have this be an Option<H256> and add a new fn to `ChainSigner` to get the `address_h256` to avoid needing to go into String and back, but I think this is probably fine